src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -33,20 +34,10 @@ private void readObject(java.io.ObjectInputStream in)
     private static final String pluginPackageName = "org.broadinstitute.hellbender.engine.filters";
     private static final Class<?> pluginBaseClass = org.broadinstitute.hellbender.engine.filters.ReadFilter.class;
 
-    @Argument(fullName = StandardArgumentDefinitions.READ_FILTER_LONG_NAME,
-            shortName = StandardArgumentDefinitions.READ_FILTER_SHORT_NAME,
-            doc="Read filters to be applied before analysis", optional=true, common = true)
-    public final List<String> userEnabledReadFilterNames = new ArrayList<>(); // preserve order
-
-    @Argument(fullName = StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME,
-            shortName = StandardArgumentDefinitions.DISABLE_READ_FILTER_SHORT_NAME,
-            doc="Read filters to be disabled before analysis", optional=true, common = true)
-    public final List<String> userDisabledReadFilterNames = new ArrayList<>();
-
-    @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
-            shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
-            doc = "Disable all tool default read filters", common = true, optional = true)
-    public boolean disableToolDefaultReadFilters = false;
+    // the purpose of this argument collection is to allow the caller to control the exposure of the command line arguments
+    @VisibleForTesting
+    @ArgumentCollection
+    public GATKReadFilterArgumentCollection userArgs;
```

Comment:
This is visible for testing but public because the test is in a different package, which is quite annoying. What do you think about moving the tests to the same package?